ci/rpm7/main/makefile

```diff
@@ -0,0 +1,79 @@
+ifndef DESTDIR
```

Comment:
Let's check in https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/devel/directoryStructure.md if some of the already existing directories would fit to include ci/ stuff (as subdirectory). If at the end we need ci/ at repo root level, the https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/devel/directoryStructure.md document should be properly changed.

Reply:
I will add changes to this doc.

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/d8f4b4b7ddb942fc7e943e053c11f057109b5b06